blocks/library/code/index.js

```diff
@@ -32,6 +32,14 @@ registerBlockType( 'core/code', {
 		},
 	},
 
+	example: {
+		content: `
+<?php
+	echo "Beep Beeb Boop";
```

Comment:
🤖

Reply:
Personally I'd find this reasonable to inline in the sense that even stylistically this is something I'd have on one line:

```js
example: {
	content: '<?php echo "Beep Beep Boop"; ?>',
},
```

Reply:
Typo: "Beeb" -> "Beep"